tests/test_coroutine.rs

```diff
@@ -245,39 +245,6 @@ fn coroutine_panic() {
     })
 }
 
-#[test]
-fn test_async_method_receiver_with_other_args() {
```

Comment:
Moved the test down as the ordering in the file made more sense to me the other way around.